hazelcast-jet-all/pom.xml

```diff
@@ -67,6 +67,25 @@
                 <configuration>
                     <createSourcesJar>true</createSourcesJar>
                     <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <artifactSet>
+                        <includes>
+                            <include>com.hazelcast.jet:hazelcast-jet-core</include>
+                            <include>com.hazelcast:hazelcast</include>
+                            <include>com.hazelcast:hazelcast-aws</include>
+                            <include>com.hazelcast:hazelcast-gcp</include>
+                            <include>com.hazelcast:hazelcast-kubernetes</include>
+                            <include>org.apache.geronimo.specs:geronimo-jms_1.1_spec</include>
+                            <include>com.rabbitmq.jms:rabbitmq-jms</include>
```

Comment:
why are these here as shaded?

Reply:
These were already shaded implicitly, I have made it explicit.

Reply:
I realized they're there because of a mistake in the core pom. Should be removed from there as well